fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,146 @@
+from fitnessllm_dataplatform.entities.dataclasses import Metrics
+from fitnessllm_dataplatform.entities.enums import (
+    FitnessLLMDataSource,
+    FitnessLLMDataStream, Status,
+)
+from fitnessllm_dataplatform.entities.queries import get_activities
+from fitnessllm_dataplatform.stream.strava.cloud_utils import get_strava_storage_path
+from fitnessllm_dataplatform.stream.strava.entities.enums import StravaStreams
+from fitnessllm_dataplatform.utils.logging_utils import logger
+
+
+def load_json_into_bq(InfrastructureNames: Enum, athlete_id: str, data_streams: list[str]):
```

Ruff (0.8.2)

Missing return type annotation for public function `load_json_into_bq`

Add return type annotation: `None`

(ANN201)